tutorials/ar-subs/index.md

```diff
@@ -0,0 +1,795 @@
+
+1. Grant also the Project Viewer role to the Service Account:
+
+        gcloud projects add-iam-policy-binding $PROJECT_ID --member serviceAccount:ml-dev@$PROJECT_ID.iam.gserviceaccount.com --role roles/viewer
+
+1. Create a JSON key for the Service Account. The key file will be downloaded to the current working directory:
+
+        gcloud iam service-accounts keys create ./credentials.json --iam-account ml-dev@$PROJECT_ID.iam.gserviceaccount.com
+
+
+### Identify your USB Microphone device number in python
+
+Next we need to identify the device number of your USB microphone visible to
+Python. Note that the device numbering for OS sound libraries may not match
+the device numbering visible to Python apps. For this reason we need to find
+the microphone device again, this time with a Python utility.
+
+1. Within the python virtual environment, execute the following command:
+
+        python3 mic_identify.py
+
+The command should output something similar to this - with the example USB
+microphone being the first entry listed here:
+
+
+    (0, 'Samson Meteor Mic: USB Audio (hw:2,0)', 2)
+    (1, 'dmix', 0)
+
+2. Note down the device number - in the above example it is: `0`
+
+
+### Test recording with the USB microphone in Python
+
+Let’s try to record audio with the identified USB microphone device,
+using `pyaudio`.
+
+1. Execute the following command. The command should record a 3 second WAV
+audio file (mono, 16bits, 44.1KHz). In this example we use the device `0` as
+identified with the previous command:
+
+        python3 mic_test.py --dev 0
+
+The command should output something similar to:
+
+
+    *** Recording 3 seconds with USB device 0 ***
+    *** Finished recording. Wrote output: test1.wav ***
+
+Note - you may get ALSA errors but you can ignore them, if the recording was
+successful.
+
+2. Listen to the test recording to make sure it worked and that the dialog
+sound quality is ok. Execute:
+
+        aplay test1.wav
+
+You may need to configure which interface the Pi uses for sound playback
+output. You can choose between the HDMI output and the 3.5mm line out /
+headphone jack, by executing: `sudo raspi-config`, and configuring the setting
+under `7: Advanced Options → A4 Audio → Choose the audio output (HDMI |
+Headphones)`.
+
+
+### Test the Media Translation API example Python client
+
+Now that your USB microphone works with Python, we can try to call the Media
+Translation API. This step will test calling the API in streaming mode, piping
+microphone audio to the service and displaying the translated live responses in
+ the command line shell.
+
+
+1. Execute the utility with the following command. You can specify the [target language code](https://cloud.google.com/translate/media/docs/languages). The default target language is `de-DE` for German. For example to test with Italian, execute the below command.
+In this example we use device number `0`. Replace that with your device number if
```

Comment:
Use 'you' instead of 'we' to address the reader, as you do in the rest of the tutorial

Reply:
Done